namespaces/collectd.yml

```diff
@@ -326,18 +426,9 @@ namespace:
     description: >
       Corresponds to collectd's df plugin.
     fields:
-    - name: space_type
```

Comment:
As for `Plugin_instance`, let's consider moving `type_instance` to the particular subgroup.